frontend/src/visualization/Graph.tsx

```diff
@@ -85,7 +86,7 @@ function Graph(props: GraphProps): JSX.Element {
 
   useObservable(
     graphDataStore.getState().pipe(
-      tap((queryResult) => {
+      tap((queryResult: QueryResult) => {
```

Comment:
TS can Inference the type automatically. It also does not change the code at runtime.
Why was the change made?